sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -74,8 +74,14 @@ public ConfigurationClient(Uri endpoint, TokenCredential credential)
         /// <param name="options">Options that allow configuration of requests sent to the configuration store.</param>
         public ConfigurationClient(Uri endpoint, TokenCredential credential, ConfigurationClientOptions options)
         {
-            Argument.AssertNotNull(endpoint, nameof(endpoint));
-            Argument.AssertNotNull(credential, nameof(credential));
+            if (endpoint == null)
```

Comment:
This is not generated code, why are changes for generator issues being made here?

Reply:
As part of the openai work we were asking remove shared source from the libraries this affects both generated and non-generated.

For extension methods we have no choice as it won't compile if the generated code uses one extension and the non generated uses another